requirement.txt

```diff
@@ -1,145 +1,62 @@
 absl-py==0.10.0
-anytree==2.8.0
-apkutils==0.8.0
-appdirs==1.4.4
 astor==0.8.1
-attrs==19.3.0
 audioread==2.1.8
-backcall==0.2.0
-bleach==3.1.5
-cachetools==4.1.1
 certifi==2020.6.20
-cffi==1.14.2
+cffi==1.14.3
 chardet==3.0.4
-cigam==0.0.3
-click==7.1.2
-cryptography==3.0
-cycler==0.10.0
-Cython==0.29.21
 decorator==4.4.2
-defusedxml==0.6.0
-distlib==0.3.1
-entrypoints==0.3
 ffmpeg-python==0.2.0
-filelock==3.0.12
-flake8 @ file:///home/conda/feedstock_root/build_artifacts/flake8_1591898334868/work
-flake8-polyfill==1.0.2
-flake8-quotes @ file:///home/conda/feedstock_root/build_artifacts/flake8-quotes_1589330264223/work
-Flask==1.1.2
 future==0.18.2
 gast==0.2.2
-google-auth==1.20.1
-google-auth-oauthlib==0.4.1
 google-pasta==0.2.0
-grpcio==1.31.0
+grpcio==1.32.0
 h5py==2.10.0
 idna==2.10
 imageio==2.9.0
 imageio-ffmpeg==0.4.2
-importlib-metadata @ file:///home/conda/feedstock_root/build_artifacts/importlib-metadata_1593211369179/work
+importlib-metadata==2.0.0
 imutils==0.5.3
-ipykernel==5.3.0
-ipython==7.16.1
-ipython-genutils==0.2.0
-itsdangerous==1.1.0
-jedi==0.17.1
-Jinja2==2.11.2
-joblib==0.16.0
-jsonschema==3.2.0
-jupyter-client==6.1.5
-jupyter-core==4.6.3
+joblib==0.17.0
 Keras-Applications==1.0.8
 Keras-Preprocessing==1.1.2
-kiwisolver==1.2.0
 librosa==0.7.2
 llvmlite==0.31.0
-Markdown==3.2.2
-MarkupSafe==1.1.1
-matplotlib==3.3.1
-mccabe==0.6.1
-mistune==0.8.4
-more-itertools @ file:///home/conda/feedstock_root/build_artifacts/more-itertools_1592197792446/work
+Markdown==3.3
```

Comment:
Which dependency is using this ?

Reply:
This comes as a dependency with Spleeter